response_operations_ui/templates/close-conversation.html

```diff
@@ -4,6 +4,7 @@
 {% from "components/fieldset/_macro.njk" import onsFieldset %}
```

Comment:
Should a user without `messages.edit` be able to access this page at all?

Reply:
No, but I felt that having a check here was worth it just in case a user somehow reaches this page without the right permissions.

Reply:
Can we have it throw the 500 error for now until the new design for the errors come through?

Reply:
The same with the other links that a user without permission should not be able to access, just have an abort(500) if they don't have the permissions